test/eval_tests/basic.js

```diff
@@ -1,10 +1,8 @@
-'use superstrict'
```

Comment:
Also, I'd like to have more tests. Try to ensure, that the plugin is working correctly.